egs/csmsc/tts1/run.sh

```diff
@@ -0,0 +1,227 @@
+#!/bin/bash
+
+# Copyright 2019 Tomoki Hayashi
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+
+# general configuration
+backend=pytorch
+stage=-1
+stop_stage=100
+ngpu=1       # number of gpus ("0" uses cpu, otherwise use gpu)
+nj=32        # numebr of parallel jobs
+dumpdir=dump # directory to dump full features
+verbose=0    # verbose option (if set > 0, get more log)
+N=0          # number of minibatches to be used (mainly for debugging). "0" uses all minibatches.
+seed=1       # random seed number
+resume=""    # the snapshot path to resume (if set empty, no effect)
+
+# feature extraction related
+fs=24000      # sampling frequency
+fmax=7600     # maximum frequency
+fmin=80       # minimum frequency
+n_mels=80     # number of mel basis
+n_fft=1024    # number of fft points
+n_shift=256   # number of shift points
+win_length="" # window length
```

Comment:
Would it be possible to change these parameters to n_fft=2048, n_shift=300, win_length=1200? (12.5ms shift, and 50ms frame length) 

These seem to be common in recent E2E TTS research using 24kHz dataset.

Reply:
I want to unify hyperparameters, but it is OK.
At least we should use the same setting for the same sampling rate.

Reply:
For 22.5k (e.g. LJSpeech):
- n_fft=1024, n_shift=256

For 24k
- n_fft=2048, n_shift=300, win_length=300  (same as Tacotron 2 paper; 12.5ms shift, 50 ms frame len)

Does this sound good for you?

Reply:
OK but I’ve never compare on 24khz, so I’m not sure this difference makes any improvement.
Do you have some experience?

Reply:
Well, I’m just following a (sort of) community standard. I guess there wouldn’t be significant difference, but if we use the same parameters with the existing method, results will be more comparable. So, if we support tacotron 2, why not using same parameters as tacotron 2?

In my limited experience, there’s no huge difference. As I mentioned in #795, training would be faster if we use larger shift.

Reply:
OK. I changed my mind.
From now, we should keep 

For 22.5k (e.g. LJSpeech):

n_fft=1024, n_shift=256
For 24k

n_fft=2048, n_shift=300, win_length=300 (same as Tacotron 2 paper; 12.5ms shift, 50 ms frame len)